core/integration/module_python_test.go

```diff
@@ -275,17 +275,33 @@ build-backend = "poetry.core.masonry.api"
 		t.Run(fmt.Sprintf("%s/%s", tc.name, tc.path), func(ctx context.Context, t *testctx.T) {
 			c := connect(ctx, t)
 
-			out, err := daggerCliBase(t, c).
-				With(fileContents("pyproject.toml", tc.conf)).
-				With(fileContents(tc.path, `
+			conf := tc.conf + fmt.Sprintf(`
+# Test: %s / %s
+`, tc.name, tc.path)
+
+			modGen := daggerCliBase(t, c).
+				With(fileContents("pyproject.toml", conf)).
+				With(fileContents(tc.path, fmt.Sprintf(`
 from dagger import function
 
+# Test: %s / %s
```

Comment:
@sipsma, maybe this is causing the test to pass by making the contents different for every subtest?

Some tests have the same `pyproject.toml`:
- setuptools
  - `src/main/__init__.py`
  - `src/main.py`
- poetry
  - `src/main/__init__.py`
  - `src/main.py`
  - `main/__init__.py`
  - `main.py`

Would be useful to know if the flake happened in other subtests or just these.

I'm going to remove the addition to the contents to see if it triggers the flake.

Reply:
Hatch failed in this run: https://dagger.cloud/dagger/traces/dfaba0c8512cff8ef05e7bee37018c82?span=819126d8189f7b28

But I wonder if the reason is because it's getting a `pyproject.toml` from one of the above or something. Just submitted a PR to merge into main. It's the same as this one, but without the `--verbose` flag in `uv`.

Reply:
Actually, `setuptools` have different contents in those subtests:

```diff
[tool.setuptools]
- packages = ["main"]
+ py-packages = ["main"]
```
So it's just `poetry` that has the same contents.